### PR TITLE
New package: mdr-0.2.4

### PR DESCRIPTION
**Pull request by jo1gi**

### CODE REVIEW

File: srcpkgs/mdr/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+version=0.2.4
+revision=1
+build_style=go
+go_import_path="github.com/MichaelMure/mdr"
+go_get="no"
+makedepends="go"
+short_desc="MarkDown Renderer for the terminal"
+maintainer="Joakim Holm <joakimholm@protonmail.com>"
+license="MIT"
+homepage="https://github.com/MichaelMure/${pkgname}"
```
Comment (abenson) [inline srcpkgs/mdr/template]: Don't use `${pkgname}` here

File: srcpkgs/mdr/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'mdr'
+pkgname=mdr
+version=0.2.4
+revision=1
+build_style=go
+go_import_path="github.com/MichaelMure/mdr"
+go_get="no"
```
Comment (abenson) [inline srcpkgs/mdr/template]: I believe this is the default.

**Comment by LinArcX** [comment]
@abenson for how long we can wait for this package?

If @jo1gi isn't ready to make changes, I can do them.

File: srcpkgs/mdr/template (hunk 2 above)
Comment (jo1gi) [reply srcpkgs/mdr/template]: I'm not sure about what I should change.

**Comment by whoizit** [comment]
@LinArcX @jo1gi you can add it any time to https://github.com/void-users/packages

**Comment by LinArcX** [comment]
@whoizit It acts like AUR? You provide binary packages?

Since, users can use xbps-src, but it needs to compile from source. if your project is different, please explain more about it.

File: srcpkgs/mdr/template (hunk 2 above)
Comment (abenson) [reply srcpkgs/mdr/template:7]: Drop this line

File: srcpkgs/mdr/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="go"
```
Comment (abenson) [inline srcpkgs/mdr/template:8]: Drop this line too

File: srcpkgs/mdr/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'mdr'
+pkgname=mdr
+version=0.2.4
```
Comment (abenson) [inline srcpkgs/mdr/template:3]: 0.2.5 is out

**Pull request closed without merging**